sdk/tables/azure-data-tables/pom.xml

```diff
@@ -40,6 +40,9 @@ Licensed under the MIT License.
       --add-opens com.azure.data.tables/com.azure.data.tables.implementation.models=ALL-UNNAMED
       --add-opens com.azure.data.tables/com.azure.data.tables.models=ALL-UNNAMED
     </javaModulesSurefireArgLine>
+    <codesnippet.skip>false</codesnippet.skip>
+    <javadocDoclet></javadocDoclet>
+    <javadocDocletOptions></javadocDocletOptions>
```

Comment:
What are the empty properties for?

Reply:
These empty lines will go away once the codesnippet tooling transition is complete. They're empty as the configuration in the parent POM is to enable the previous codesnippet doclet and this empty configuration disables it.